arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -1151,1137 +1151,4913 @@
+        "description": "Creates or updates an integration account certificate.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateName",
+            "description": "The integration account certificate name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "certificate",
+            "description": "The integration account certificate.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountCertificate"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountCertificate"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/IntegrationAccountCertificate"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationAccountCertificates"
+        ],
+        "operationId": "IntegrationAccountCertificates_Delete",
+        "description": "Deletes an integration account certificate.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "The resource group name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "integrationAccountName",
+            "description": "The integration account name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "certificateName",
+            "description": "The integration account certificate name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      }
     }
   },
   "definitions": {
     "Resource": {
```

Comment:
@pankajsn There is an issue https://github.com/Azure/azure-rest-api-specs/issues/898 reported by a customer which is _correct_. Can you please resolve this issue, by accurately representing the properties of "Resource" `"readOnly": true`.

- "id", "name" and "type" should be marked `"readOnly": true` as per ARM guidelines.
- Shouldn't "location" be a required property `"required": ["location"]`?